readthedocs/settings/base.py

```diff
@@ -65,6 +65,11 @@ class CommunityBaseSettings(Settings):
     CSRF_COOKIE_HTTPONLY = True
     CSRF_COOKIE_AGE = 30 * 24 * 60 * 60
 
+    # Encryption
+    FIELD_ENCRYPTION_KEY = [
+        'P98DEYWoSuLmgXdRGzJkbo1JWeiHf4ghpFI8QvV3hRI=',  # replace this please
```

Comment:
Hrm, not sure we should provide a default here. Maybe only on dev settings? I don't want to ship with a key that will be reused by default, as I'm sure most private instances would gloss over this and just use the default key.

Reply:
Makes sense.

I will move this to `dev.py` file and create the proper PRs into the operations repositories.

Reply:
https://github.com/rtfd/readthedocs-corporate-ops/pull/107